housing_django/static/js/controlview.js

```diff
@@ -3,9 +3,9 @@ var ControlView = Backbone.View.extend({
     this.applicantForm = options.applicantForm;
   },
   events: {
-    'click .show-form': 'showForm'
+    'click .new-applicant': 'newApplicant'
   },
-  showForm: function() {
+  newApplicant: function() {
```

Comment:
since this is a method and not a property, can you begin the file name with a verb? I.e. showNewApplicant